### PR TITLE
Doc/ontrack comprehensive setup

### PR DESCRIPTION
**Pull request by lachfoy**
# Description

Added a new guide for setting up OnTrack on Windows using WSL2. This was mostly done to help myself as I was getting confused with the information being in so many different places at once, and having some outdated tutorials on setting up.

### CODE REVIEW

File: docs/learning/training/markdown-guide.md [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 ## What is Markdown?
 
-Markdown (md) is a markup language, similar to a .txt file, but with basic **in-text** formatiing
+Markdown (md) is a markup language, similar to a .txt file, but with basic **in-text** formating
```
Comment (ghost) [inline docs/learning/training/markdown-guide.md:17]: Nice catch

Comment (lachfoy) [reply docs/learning/training/markdown-guide.md:17]: Actually it should be _formatting_!! My correction wasn’t even spelled correctly…

**Comment by ghost** [comment]
@lachfoy are you also able to run the formatter on the guide.
either through npm or the prettier extension in VS Code? Currently the file will not merge (after review) due to formatting issues.

**Comment by lachfoy** [comment]
> @lachfoy are you also able to run the formatter on the guide. either through npm or the prettier extension in VS Code? Currently the file will not merge (after review) due to formatting issues.

Done!

File: docs/learning/training/ontrack-comprehensive-setup.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+cd Doubtfire
+```
+
+clone your doubtfire-deploy
+
+```sh
+git clone --recurse-submodules https://github.com/YOUR_USERNAME/doubtfire-deploy
+```
+
+WIP!!!!!
```
Comment (tancnle) [inline docs/learning/training/ontrack-comprehensive-setup.md:70]: **question:** @lachfoy is this PR still a draft? Are you still working on these content?

Comment (lachfoy) [reply docs/learning/training/ontrack-comprehensive-setup.md:70]: @tancnle Yes - It is a work in progress. Should I instead finalize before creating a PR?

Comment (tancnle) [reply docs/learning/training/ontrack-comprehensive-setup.md:70]: Thanks for clarifying @lachfoy. 

> Should I instead finalize before creating a PR?

Not really. According to our code review [guideline](https://github.com/thoth-tech/handbook/blob/main/docs/processes/quality-assurance/git-contribution-guide.md#code-review-guidelines), we would prefer to have a **draft PR** created as soon as you can. I would suggest setting this as a Draft so people know that you are still working on it.

Comment (lachfoy) [reply docs/learning/training/ontrack-comprehensive-setup.md:70]: Ah, sorry! I'm still pretty new to GitHub still, and I actually had no idea that was a thing. I will do that.

**Pull request closed without merging**